crates/router/src/types.rs

```diff
@@ -891,6 +894,40 @@ impl From<&&mut PaymentsAuthorizeRouterData> for ConnectorCustomerData {
     }
 }
 
+impl<F> From<&RouterData<F, PaymentsAuthorizeData, PaymentsResponseData>>
+    for PaymentMethodTokenizationData
+{
+    fn from(data: &RouterData<F, PaymentsAuthorizeData, PaymentsResponseData>) -> Self {
+        Self {
+            payment_method_data: data.request.payment_method_data.clone(),
+            browser_info: None,
+            currency: data.request.currency,
+            amount: data.request.amount,
+        }
+    }
+}
```

Comment:
If it is not used anyewhere, please remove it

Reply:
Removed